wasm/main.go

```diff
@@ -0,0 +1,62 @@
+//go:build js && wasm
+
+package main
+
+import (
+	"crypto/ed25519"
+	"encoding/base64"
+	"encoding/json"
+	"syscall/js"
+
+	"github.com/TBD54566975/ssi-sdk/crypto"
+	"github.com/TBD54566975/ssi-sdk/did"
+)
+
+/*
+ * This is the glue to bind the functions into javascript so they can be called
+ */
+func main() {
+	done := make(chan struct{})
```

Comment:
curious what the difference is if this channel is removed?

Reply:
IIRC, you need to keep the process alive.

Reply:
✅